src/main/java/seedu/address/ui/InfoPanel.java

```diff
@@ -51,49 +61,80 @@
     private Label salaryOwed;
     @FXML
     private FlowPane tags;
+    @FXML
+    private Label textOverlay;
 
     /**
      * Creates a {@code InfoPanel} with the given {@code Person}.
      * @param p
      */
     public InfoPanel(ObservableObjectValue<Person> p) {
         super(FXML);
+
+        textOverlay.setText(LIST_EMPTY_MSG);
+        textOverlay.setTextAlignment(TextAlignment.CENTER);
+
         updateInfoPanel(p.get());
         p.addListener((x, y, z) -> {
             updateInfoPanel(z);
             System.out.println(y);
         });
     }
 
+    /**
+     * Removes the overlay on top of the InfoPanel
```

Comment:
Just a suggestion, but maybe it might be better to specify what the overlay is in the comments? Such as how it is used to display a message above the employee details or something like that?

Reply:
Added further explanation on what the overlay has.